packages/ai/src/ui/convert-to-model-messages.ts

```diff
@@ -149,12 +149,7 @@ export function convertToModelMessages(
               } else if (part.type === 'dynamic-tool') {
                 const toolName = part.toolName;
 
-                if (part.state === 'input-streaming') {
-                  throw new MessageConversionError({
-                    originalMessage: message,
-                    message: `incomplete tool input is not supported: ${part.toolCallId}`,
-                  });
-                } else {
+                if (part.state !== 'input-streaming') {
```

Comment:
how are the changes in this file and chat.ts above related to the issue? can these changes be separated?

Reply:
i wonder how to best handle/document this. right now from what i understand input-streaming would be dropped automatically, correct?